mobile/android/android-device.ts

```diff
@@ -173,6 +173,10 @@ export class AndroidDevice implements Mobile.IAndroidDevice {
 		return (() => {
 			var startPackageCommand = this.composeCommand("shell am start -a android.intent.action.MAIN -n %s/%s", packageName, this.$staticConfig.START_PACKAGE_ACTIVITY_NAME);
 			this.$childProcess.exec(startPackageCommand).wait();
+
+			if (options.logging) {
```

Comment:
`--appLog`, `--appOutput`, `--outputAppLog`, `--streamAppLog`, `--streamAppOutput` are all viable options.

I am more inclined towards one of those: `--outputAppLog`, `--streamAppLog`, `--streamAppOutput` as they are more intuitive and self-explanatory => minimal room for confusion.